code/datums/abilities/critter/gibstare.dm

```diff
@@ -38,15 +45,20 @@
 
 	onEnd()
 		..()
+		if(istype(owner, /obj/critter))
+			var/obj/critter/ownerCritter = owner
+			if (!ownerCritter.alive)
+				interrupt(INTERRUPT_ALWAYS)
+				return
 		var/mob/ownerMob = owner
```

Comment:
Redundant var, should just use `owner` (especially since it's not always a mob but sometimes a critter).